docs/user/kafka/index_config.rst

```diff
@@ -38,6 +38,11 @@ parameter. When a producer writes an update to an existing feature, the consumer
 Once the timeout is hit without any updates, the feature will be removed from the consumer cache and will no
 longer be returned when querying.
 
+For advanced use cases, the ``kafka.cache.expiry.dynamic`` data store parameter can be used to expire features
+based on matching filter predicates. The value should be a TypeSafe config document where the keys are CQL filter
+strings and the values are expiration duration strings (for example, ``{ "type = 'boat'": "100ms" }``). Any
+features which do not match one of the filters will use the default ``kafka.cache.expiry`` value, if provided.
```

Comment:
Any particular reason to use TypeSafe config?

Reply:
originally i used json, but typesafe config will parse json and it's a lot more flexible wrt injecting sys properties, syntax, etc

Reply:
System properties?  We can inject system properties into the TypeSafe config for this feature?!

Reply:
https://github.com/lightbend/config#optional-system-or-env-variable-overrides